elephant/kernels.py

```diff
@@ -142,6 +146,13 @@ def __init__(self, sigma, invert=False):
         if not isinstance(sigma, pq.Quantity):
             raise TypeError("'sigma' must be a quantity")
 
+        if sigma.ndim > 0:
+            # handle the one-dimensional case of size 1
+            if sigma.ndim == 1 & sigma.size == 1:
```

Comment:
Just a side remark: 
This will also catch something like `sigma = [[5]] * pq.s`
Seems fine to me though, since a meaningful error is raised in that case now.